### DISCUSSION TITLE
Publishing v3 alpha

### DISCUSSION THREAD

**Discussion by meatballhat** [urfave/cli#1507] [General]
I would like to add and push a `v3.0.0-alpha` tag in the `v3-dev-main` branch in the near future. This will allow projects to begin trying out the `v3` API [as described here](https://go.dev/doc/modules/release-workflow#pre-release).

```bash
go get github.com/urfave/cli/v3@v3.0.0-alpha
```

I realize that the `v3` API is currently very similar to the `v2` API, but larger breaking changes are coming soon. Please let us know what concerns or questions you may have!

![three tarsiers](https://upload.wikimedia.org/wikipedia/commons/thumb/b/ba/Bohol.tarsier_jtlimphoto.JPG/800px-Bohol.tarsier_jtlimphoto.JPG)

**Comment by chkohner** [comment]
Is there a document outlining the design goals of v3? There are certainly a fair number of pain points we've been dealing with on v2, and it'd be nice to see if those are under consideration.

**Reply by dearchap** [reply]
See #833
